app/templates/package.json

```diff
@@ -1,23 +1,25 @@
 {
   "name": "<%= projectName %>",
   "description": "BEM project",
-  "version": "0.0.0",
+  "version": "1.0.0",
   "author": {
     "name": "<%= author %>",
     "email": "<%= email %>"
   },
   "repository": {},
   "engines": {
-    "node": ">=0.10.0"
+    "node": ">=4.0.0"
+  },
+  "dependencies": {
+    "bower": "^1.7.9"
```

Comment:
If you want `bower` as a dependency, but not as devDependency (as it is implemented now) you should remove it from [here](https://github.com/bem/generator-bem-stub/blob/master/app/lib/enb.js#L74).

Because in current implementation of your pull request `bower` will be added to `dependencies` and `devDependencies` both , so `npm` logs warning:

![s3j2j-2016-05-12_16-40-59](https://cloud.githubusercontent.com/assets/6376693/15216590/5f9f06b8-1860-11e6-9fdb-6e676c7952ba.png)